addons/mail/static/src/discuss/core/common/notification_settings.js

```diff
@@ -17,4 +17,12 @@ export class NotificationSettings extends Component {
         this.store.settings.setMuteDuration(minutes, this.props.thread);
         this.props.close?.();
     }
+
+    onClickServerMuted() {
+        this.env.services.action.doAction({
+            tag: "mail.discuss_notification_settings_action",
+            type: "ir.actions.client",
+            target: "new",
+        });
+    }
```

Comment:
On public page, the server is mute but the action service is not available , resulting in crash when clicking on it

![Screenshot 2024-08-27 at 13 16 34](https://github.com/user-attachments/assets/fe8d7223-b77b-4001-9598-12693ed3ab61)

Reply:
I removed the `doAction` and built a custom modal. Now it works as expected